process/interface.go

```diff
@@ -340,6 +340,7 @@ type IntermediateProcessorsContainerFactory interface {
 
 // VirtualMachinesContainer defines a virtual machine holder data type with basic functionality
 type VirtualMachinesContainer interface {
+	Closer
```

Comment:
add the Close function directly to the interface.

Reply:
Fixed.